client/components/repository/index.vue

```diff
@@ -34,6 +34,31 @@ import selectActivity from '@/components/repository/common/selectActivity';
 import sortBy from 'lodash/sortBy';
 import withUserTracking from 'components/common/mixins/userTracking';
 
+const getTabItems = ({ hasWorkflow, isAdmin, isRepositoryAdmin, hasActivities }) => {
+  return [
```

Comment:
Consider writing it as arrow func:
```
const getTabItems = ({ hasWorkflow, isAdmin, isRepositoryAdmin, hasActivities }) => [
  { ....
```
To create slightly leaner function

Reply:
Done.